R/shiny-getters.R

```diff
@@ -79,31 +80,29 @@
 #'     [addEventListener()](https://github.com/vega/vega/tree/master/packages/vega-view#view_addEventListener)
 #' @export
 #'
-vw_shiny_get_signal <- function(outputId, name, body_value = "value") {
+vw_shiny_get_signal <-
+  function(outputId, name, body_value = "value", id = NULL) {
 
   assert_packages("shiny")
 
   session <- shiny::getDefaultReactiveDomain()
-
-  inputId <- ""
+  ns <- shiny::NS(id)
```

Comment:
I know this is a picky one, but I would put the `shiny::NS()` call first, before the `shiny::getDefaultReactiveDomain()` call.